packages/components/inputs/search-select-input/src/messages.js

```diff
@@ -0,0 +1,15 @@
+import { defineMessages } from 'react-intl';
```

Comment:
The `search-select-input` will have different default message to those that are being used by the rest of the select inputs/fields. Therefore they were added here